module-2/newspaper/index.html

```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>The Guardian</title>
+    <link rel="stylesheet" href="styles.css">
+    <style>
+        html, body {
+            height: 110%;
+        }
+        body {
+            display: flex;
+            flex-direction: column;
+            align-items: center;
+            justify-content: center;
+            margin: 0;
+            overflow-y: auto;
+            background-color: rgb(237, 237, 237);
+        }
+        header {
+            display: flex;
+            flex-direction: row;
+            justify-content: flex-end;
+            width: 100%;
+            padding: 20px;
+        }
+        .side-header {
+            display: flex;
+            flex-direction: column;
+            justify-content: start;
+            width: 100%;
+            padding: 20px;
+        }
+     
+        .main-content {
+            display: flex;
+            flex-direction: row;
+            align-items: center;
+            justify-content: center;
+            gap: 20px;
+            margin: 20px;
+            
+        }
+        .main-content2 {
+            display: flex;
+            flex-direction: row;
+            align-items: center;
+            justify-content: center;
+            gap: 20px;
+            margin: 20px;
+            
+        }
+        footer {
+            width: 100%;
+            text-align: center;
+            padding: 10px;
+            background-color: lightgray;
+        }
+        .row img {
+            width: auto;;
+            height: auto;
+
+        }
```

Comment:
These styles should be in your `styles.css` file, not in the head. While it will work this way, but is not a pattern you'd want to adopt for larger/professional projects